### PR TITLE
Fix example files

### LINKED ISSUE #41: Missing class call in example files
**Summary**
The first line of Cell 17 (or line 55) of the example file "basic.ipynb" omits calling the "model" class. Should be modified to:

test2scen = sansmic.model.Scenario(

The same omission happens multiple times in the example file.

Thanks!

Screenshot 2024-10-02 151903

### PR DESCRIPTION
**Pull request by dbhart**
Resolves #41. The examples had errors that were not caught during the CI testing. In addition to fixing the errors, the CI testing was improved so that these errors could be caught in the future.

### CODE REVIEW

File: .github/workflows/continuous-integration.yml
Comment (dbhart) [inline .github/workflows/continuous-integration.yml:1]: Merged jobs into single job

File: .github/workflows/continuous-testing.yml
Comment (dbhart) [inline .github/workflows/continuous-testing.yml:1]: Added a fast test on a single os and version for every commit anywhere

File: .github/workflows/release.yml
Comment (dbhart) [inline .github/workflows/release.yml:1]: White space cleaned by pre commit hook

File: .pre-commit-config.yaml
Comment (dbhart) [inline .pre-commit-config.yaml:1]: Add cleanser to strib binary objects from nbs

File: examples/baseline.json
Comment (dbhart) [inline examples/baseline.json:1]: Was missing the baseline data to test against

File: examples/basic.ipynb
Comment (dbhart) [inline examples/basic.ipynb:1]: Main fixes occurred in this file. Module paths were incorrect in notebook

File: examples/regression.ipynb
Comment (dbhart) [inline examples/regression.ipynb:1]: Main fixes occur in this file. Module paths were incorrect and baseline data was missing from comparisons

File: pyproject.toml
Comment (dbhart) [inline pyproject.toml:1]: add matplotlib for testing notebooks

File: requirements.txt
Comment (dbhart) [inline requirements.txt:1]: need matplotlib and nbmake to test notebooks

**Review by tzeitle — APPROVED** [review]
Confirmed that tests run correctly in Python notebooks.

**Pull request merged**